permissions/plugin-line-notify.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "line-notify"
+github: "jenkinsci/line-notify-plugin"
+paths:
+- "org/jenkins-ci/plugins/line-notify"
```

Comment:
Inconsistent with https://github.com/jenkinsci/line-notify-plugin/blob/1cd4eaddde5633c32a5a38aca7a137b57d9e215c/pom.xml#L10, please see this repository's readme for instructions.